examples/api/lib/router.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Api.Router do
+  use Francis
+  alias Api.Todos
+
+  # List all todos
+  get("/todos", fn _ ->
+    Todos.list_todos()
+  end)
+
+  # Get a single todo by ID
+  get("/todos/:id", fn %{params: %{"id" => id}} ->
+    try do
+      Todos.get_todo!(String.to_integer(id))
+    rescue
+      Ecto.NoResultsError -> {:status, 404, %{error: "Not found"}}
+    end
+  end)
+
+  # Create a new todo
+  post("/todos", fn %{body: attrs} ->
```

Comment:
Shouldn't be body_params? I would never the less just use "params" (like the merge from query params and body params).